main.go

```diff
@@ -64,35 +71,90 @@ func init() {
 }
 
 func main() {
-	err := startOperator()
-	if err != nil {
-		setupLog.Error(err, "failed to start operator")
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	if len(os.Args) < 2 {
+		fmt.Printf("expected one of the following subcommands:\n"+
+			"  %s - start in operator-mode, reconciling this operators custom resources\n"+
+			"  %s - start in garbage-collection mode, deleting backups according to the configured retention strategy\n",
+			operatorCmd.Name(),
+			garbageCollectorCmd.Name())
 		os.Exit(1)
 	}
+
+	switch os.Args[1] {
+	case operatorCmd.Name():
+		err := startOperator(ctx, operatorCmd, os.Args[2:])
+		if err != nil {
+			setupLog.Error(err, "failed to start operator")
```

Comment:
the setupLog is not visible in the logs of the container, so that is difficult to see what the actual error is

Reply:
Done, I replaced it.